evm/test/Oracle.test.ts

```diff
@@ -867,8 +867,8 @@ describe('Oracle', () => {
           )
           const receipt = await tx.wait()
 
-          assert.equal(receipt.logs!.length, 2)
-          assert.equal(request.id, receipt.logs![0].topics[1])
+          assert.equal(receipt.logs?.length, 2)
+          assert.equal(request.id, receipt.logs?.[0].topics[1])
```

Comment:
@HenryNguyen5 Why doesn't typescript complain about `.topics[1]`? `logs?.[0]` marks that it could be undefined, yet further indexing works without any errors.